pkg/identityserver/store.go

```diff
@@ -24,15 +24,16 @@ import (
 	"go.thethings.network/lorawan-stack/v3/pkg/log"
 )
 
-var bunstoreFeatureFlag = experimental.DefineFeature("is.bunstore", false)
+var gormFeatureFlag = experimental.DefineFeature("is.gormstore", false)
 
 func (is *IdentityServer) setupStore() error {
-	if bunstoreFeatureFlag.GetValue(is.Context()) {
-		return is.setupBunStore()
+	if gormFeatureFlag.GetValue(is.Context()) {
+		return is.setupGormStore()
 	}
-	return is.setupGormStore()
+	return is.setupBunStore()
 }
 
+// TODO: #5643 Remove the gormstore implementation.
```

Comment:
In general when we use these `TODO` tags we link the issue instead of using the number.